tests/components/modern_forms/test_light.py

```diff
@@ -0,0 +1,143 @@
+"""Tests for the Modern Forms light platform."""
+from unittest.mock import patch
+
+from aiomodernforms import ModernFormsConnectionError
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, DOMAIN as LIGHT_DOMAIN
+from homeassistant.components.modern_forms.const import (
+    ATTR_SLEEP_TIME,
+    DOMAIN,
+    SERVICE_CLEAR_LIGHT_SLEEP_TIMER,
+    SERVICE_SET_LIGHT_SLEEP_TIMER,
+)
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_ON,
+    STATE_UNAVAILABLE,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from tests.components.modern_forms import init_integration
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+
+async def test_light_state(
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
+) -> None:
+    """Test the creation and values of the Modern Forms lights."""
+    await init_integration(hass, aioclient_mock)
+
+    entity_registry = er.async_get(hass)
+
+    state = hass.states.get("light.modernformsfan_light")
+    assert state
+    assert state.attributes.get(ATTR_BRIGHTNESS) == 128
```

Comment:
Consider testing all the attributes here to make sure it's as expected:

assert state.attributes == {ATTR_BRIGHTNESS: 128, ...}

Reply:
added in check on friend name based on return from rest call.  Its the only other thing in the state attributes that gets set dynamically from the call.